pkg/tagger/types/types.go

```diff
@@ -77,3 +78,9 @@ type EntityEvent struct {
 	EventType EventType
 	Entity    Entity
 }
+
+// TagsBuilder represents a consumer for tags stored in the tagger.
```

Comment:
In a broad sense it is similar to producer/consumer - we give an object to the tagger, and it calls a method on it to feed it the data. As for the name, yes, it could. But this interface replaces a pre-existing type, and I think it is helpful to keep the same name, to preserve knowledge that people have about the code.

Reply:
Hm, I'm not sure I agree with that reasoning, as it biases the code for understanding by people who already understand it, at the cost of everyone else.. But, OK :)